Update definition files and add more languages

kaskii opened the pull request:
Found definition files in Daikin diagnostic tool and wrote a small program to convert all of them into ESPAltherma format. 

I'll open this as a draft for comments. 

TODO:

- ~~Uncomment some values in default.h files.~~

kaskii commented:
Note that this is a dump of Daikin diagnostic tool definition files, I can not verify how accurate the translations are.

raomin commented:
Thanks for this @kaskii. Looks like you got an updated version of dchecker. Good! I'll try to get it and have a look if there are some other interesting stuff...

On your PR, can you please:
1. Normalize the file names `Altherma` -> `ALTHERMA`
2. Keep the default.h untouched (or uncomment the exposed values) as the diffs are minimal in existing files.

Thanks again (and thanks for the beers!) 👍

kaskii commented:
Includes all dumps ("I protocol") from D-Checker v. 3.7.0.6.

Added missing translations for various languages and heat pumps. Also added new languages:
- Italian
- Japanese

New pumps:
- ALTHERMA(EBLA-EDLA  EWAA-EWYA D SERIES 9-16KW)
- ALTHERMA(LT_GAS_INJ)
- ALTHERMA(TOP-GRADE-LARGE_E)TEST
- ALTHERMA(TOP-GRADE-SMALL_E)TEST
- EKHWET-BAV3(MULTI DHW TANK)
